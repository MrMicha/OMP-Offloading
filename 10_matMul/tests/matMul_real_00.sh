#!/bin/bash
#CCS -N matMul
#CCS -t 10m
#CCS -g pc2-mitarbeiter
#CCS --res=rset=1:gtx1080=1,place=:excl

../src/matMul $((2**12))
