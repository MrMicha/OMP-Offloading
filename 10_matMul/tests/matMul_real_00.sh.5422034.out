hallo from gpu011
matrix dim: 4096 x 4096
time averaged over 16 loops
matMulAB (0) :      24.7 GFLOPS      25.3 GFLOPS maxabserr =       0.0
matMulAB (1) :       9.8 GFLOPS       9.9 GFLOPS maxabserr =       0.0
matMulAB (2) :     179.0 GFLOPS     220.7 GFLOPS maxabserr =       0.0
matMulAB (3) :       5.1 GFLOPS       5.1 GFLOPS maxabserr =    1028.8
matMulAB (4) :     171.2 GFLOPS     209.0 GFLOPS maxabserr =       0.0
matMulAB (5) :     342.0 GFLOPS     533.1 GFLOPS maxabserr =       0.0
matMulAB (6) :     634.9 GFLOPS    1717.8 GFLOPS maxabserr =       0.0
matMulAB (7) :     217.8 GFLOPS     282.3 GFLOPS maxabserr =       0.0
matMulAB (8) :     236.0 GFLOPS     313.6 GFLOPS maxabserr =       0.0
matMulAB (9) :     341.6 GFLOPS     534.8 GFLOPS maxabserr =       0.0
matMulAB (10) :     988.8 GFLOPS   10645.0 GFLOPS maxabserr =       0.0
